openpype/hosts/blender/plugins/publish/extract_layout.py

```diff
@@ -131,43 +129,32 @@ def process(self, instance):
 
         fbx_count = 0
 
+        project_name = instance.context["projectEntity"]["name"]
```

Comment:
I am getting this error:

```
Traceback (most recent call last):
  File "C:\Dev\OpenPype\openpype\.venv\lib\site-packages\pyblish\plugin.py", line 791, in __getitem__
    return next(c for c in self if c.id == item)
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Dev\OpenPype\openpype\.venv\lib\site-packages\pyblish\plugin.py", line 522, in __explicit_process
    runner(*args)
  File "C:\Dev\OpenPype\openpype\openpype\hosts\blender\plugins\publish\extract_layout.py", line 132, in process
  File "C:\Dev\OpenPype\openpype\.venv\lib\site-packages\pyblish\plugin.py", line 793, in __getitem__
    raise KeyError("%s not in list" % item)
KeyError: 'projectEntity not in list'
```

I think this should be `instance.data["projectEntity"]["name"]`?

Reply:
Should be fixed